AEP/aep-xxxxx.md

```diff
@@ -0,0 +1,245 @@
+
+```tf
+    "Components__QueueAdaptorSettings__ClassName"           = "ArmoniK.Core.Adapters.Amqp.QueueBuilder"
+    "Components__QueueAdaptorSettings__AdapterAbsolutePath" = "/adapters/queue/amqp/ArmoniK.Core.Adapters.Amqp.dll"
+    "Amqp__User"                                            = ""
+    "Amqp__Password"                                        = ""
+    "Amqp__Host"                                            = ""
+    "Amqp__Port"                                            = ""
+    "Amqp__Scheme"                                          = "AMQP"
+    "Amqp__MaxPriority"                                     = ""
+    "Amqp__MaxRetries"                                      = ""
+    "Amqp__LinkCredit"                                      = ""
+
+```
+The internal module will then acts as a consumer of the various outputs from external modules. 
+
+## ArmoniK inputs (from storage module outputs)
+
+```tf
+variable "control_plane" {
+  type = object({
+    // ...
+    conf = optional(list(object({
+      env = optional(map(string), {})
+      env_configmap = optional(set(string), [])
+      env_from_configmap = optional(map(object({
+        configmap = string
+        field  = string
+    })), {})
+    })), [])
+  })
+}
+
+variable "compute_plane" {
+  type = object({
+    // ...
+    conf = optional(list(object({
+      env = optional(map(string), {})
+      env_configmap = optional(set(string), [])
+      env_from_configmap = optional(map(object({
+        configmap = string
+        field  = string
+    })), {})
+    })), [])
+  })
+}
+
+variable "polling_agent" {
+  type = object({
+    // ...
+    conf = optional(list(object({
+      env = optional(map(string), {})
+      env_configmap = optional(set(string), [])
+      env_from_configmap = optional(map(object({
+        configmap = string
+        field  = string
+    })), {})
+    })), [])
+  })
+}
+
+variable "worker" {
+  type = object({
+    // ...
+    conf = optional(list(object({
+      env = optional(map(string), {})
+      env_configmap = optional(set(string), [])
+      env_from_configmap = optional(map(object({
+        configmap = string
+        field  = string
+    })), {})
+    })), [])
+  })
+}
+
+variable "metric_exporter" {
+  type = object({
+    // ...
+    conf = optional(list(object({
+      env = optional(map(string), {})
+      env_configmap = optional(set(string), [])
+      env_from_configmap = optional(map(object({
+        configmap = string
+        field  = string
+    })), {})
+    })), [])
+  })
+}
+```
+
+
+Three components are part of the Armonik Core and are those consuming the outputs. 
+
+control-plane
+
+compute-plane
+- polling-agent
+- worker
+
+metrics-exporter
+
+## Component outputs
+
+Component outputs will be something like that:

```

Comment:
This sentence should be more precise. This AEP is there to normalize it. "something like" leave too much freedom and interpretation.